scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/config/SimpleUsersConfig.java

```diff
@@ -9,6 +9,7 @@
 
     @Override
     public void configure(SimpleUsers simpleUsers) {
-        
+     simpleUsers.resetRequestSentEndpoint("/reset/sent");
```

Comment:
there should only be one config class for simple users; either this one or the `EmailConfig` one.